source/includes/wp-api-v1/_products.md

```diff
@@ -155,7 +155,7 @@ The products API allows you to create, view, update, and delete individual, or a
 | `download_expiry`    | integer   | Number of days that the customer has up to be able to download the variation, the `-1` means that **downloads never expires**. Default is `-1`.                                                                       |
 | `tax_status`         | string    | Tax status. Default is `taxable`. Options: `taxable`, `shipping` (Shipping only) and `none`.                                                                                                                          |
 | `tax_class`          | string    | Tax class.                                                                                                                                                                                                            |
-| `manage_stock`       | boolean   | Stock management at variation level. Default is `false`.                                                                                                                                                              |
+| `manage_stock`       | boolean, string   | Stock management at variation level. Default is `false`.                                                                                                                                                              |
```

Comment:
Can we go ahead and note in the description that this either a boolean or `parent` since there are no other string options?